Resources/input/fsh/profiles/0_GEM_PR_KIM_MessageHeader.fsh

```diff
@@ -0,0 +1,30 @@
+Profile: GEM_PR_KIM_MessageHeader
+Parent: MessageHeader
+Id: GEM-PR-KIM-MessageHeader
+Title: "GEM PR KIM MessageHeader"
+Description: "Basic Message Header for communication of FHIR Ressources with KIM Messages"
+* ^status = #draft
+
+//
+* event[x] from GEM_VS_MEDREQ_KIM_Dienstkennung (required)
+//sender und receicer als Identifier und KIM-Adresse als Identifier modellieren
+// Im FHIR Message header ist source.endpoint und destination.endpoint 1..1, soll das als KIM Adresse genutzt werden?
+//email ist uri?
+//refernzpunkt display als Anzeigename
+* sender 1..1 MS
+* sender.identifier 1..1 MS
+* sender.identifier only GEM_PR_KIM_AdressIdentifier or GEM_PR_TIM_AdressIdentifier
+* sender.display 0..1 MS
+  * ^short = "Anzeigename der KIM Adresse"
+* destination 1..* MS
+* destination.receiver 1..1
+* destination.receiver.identifier 1..1 MS
+* destination.receiver.identifier only GEM_PR_KIM_AdressIdentifier or GEM_PR_TIM_AdressIdentifier
+* destination.receiver.display 0..1 MS
+  * ^short = "Anzeigename der KIM Adresse"
+* focus 1..*
+
+//MS für alle elemente
+
+//Was passiert auf KIM EBNE?
+//Wie gehen wir damit um, wenn FHIR nicht valide
```

Comment:
@simoneOnFhir Hast du eine Idee, wie wir damit umgehen sollten, falls die übermittelten FHIR Ressourcen nicht valide sind?

Reply:
Wenn ein Empfänger die Nachricht nicht verarbeiten kann (egal ob die Nachricht nicht valide ist oder ob es einen Verarbeitungsfehler gab), müsste ein NACK gesendet werden, also eine Message mit einem entsprechenden Eventcode für Fehlermeldungen (gibt es den in KIM?) und als Focus eine OperationOutcode-Ressource mit Details zum Fehler.